src/TagLib/Id3v2/Frames/UrlLinkFrame.cs

```diff
@@ -0,0 +1,834 @@
+	/// </value>
+	/// <remarks>
+	///    This value will be overwritten if <see
+	///    cref="TagLib.Id3v2.Tag.ForceDefaultEncoding" /> is <see
+	///    langword="true" />.
+	/// </remarks>
+	public StringType TextEncoding
+	{
+	  get
+	  {
+		ParseRawData();
+		return encoding;
+	  }
+	  set { encoding = value; }
+	}
+
+#endregion
+
+#region Public Methods
+
+	/// <summary>
+	///    Gets a string representation of the current instance.
+	/// </summary>
+	/// <returns>
+	///    A <see cref="string" /> containing the joined text.
+	/// </returns>
+	public override string ToString()
+	{
+	  ParseRawData();
+	  return string.Join("; ", Text);
+	}
+
+	/// <summary>
+	///    Renders the current instance, encoded in a specified
+	///    ID3v2 version.
+	/// </summary>
+	/// <param name="version">
+	///    A <see cref="byte" /> value specifying the version of
+	///    ID3v2 to use when encoding the current instance.
+	/// </param>
+	/// <returns>
+	///    A <see cref="ByteVector" /> object containing the
+	///    rendered version of the current instance.
+	/// </returns>
+	public override ByteVector Render(byte version)
+	{
+	  return base.Render(version);
+	}
+
+#endregion
+
+#region Public Static Methods
+
+	/// <summary>
+	///    Gets a <see cref="UrlLinkFrame" /> object of a
+	///    specified type from a specified tag, optionally creating
+	///    and adding one with a specified encoding if none is
+	///    found.
+	/// </summary>
+	/// <param name="tag">
+	///    A <see cref="Tag" /> object to search for the specified
+	///    tag in.
+	/// </param>
+	/// <param name="ident">
+	///    A <see cref="ByteVector" /> object containing the frame
+	///    identifer to search for.
+	/// </param>
+	/// <param name="create">
+	///    A <see cref="bool" /> value specifying whether or not to
+	///    create a new frame if an existing frame was not found.
+	/// </param>
+	/// <returns>
+	///    A <see cref="UrlLinkFrame" /> object containing
+	///    the frame found in or added to <paramref name="tag" /> or
+	///    <see langword="null" /> if no value was found <paramref
+	///    name="create" /> is <see langword="false" />.
+	/// </returns>
+	/// <remarks>
+	///    To create a frame without having to specify the encoding,
+	///    use <see cref="Get(Tag,ByteVector,bool)" />.
+	/// </remarks>
+	/// <exception cref="ArgumentNullException">
+	///    <paramref name="tag" /> or <paramref name="type" /> is
+	///    <see langword="null" />.
+	/// </exception>
+	/// <exception cref="ArgumentException">
+	///    <paramref name="type" /> is not exactly four bytes long.
+	/// </exception>
+	public static UrlLinkFrame Get(Tag tag,
+											ByteVector ident,
+											bool create)
+	{
+	  if (tag == null)
+		throw new ArgumentNullException("tag");
+
+	  if (ident == null)
+		throw new ArgumentNullException("ident");
+
+	  if (ident.Count != 4)
+		throw new ArgumentException(
+		  "Identifier must be four bytes long.",
+		  "ident");
```

Comment:
instead of doing "ident" or "tag", use "nameof (tag)" or "nameof(ident)". This way if the parameters are renamed we don't need to worry about updating the strings in these lines as well.